src/scripts/install-cli.sh

```diff
@@ -20,7 +20,7 @@ if [ "$(id -u)" -ne 0 ]; then
 fi
 
 # Install op-cli
-if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+if [[ "$OSTYPE" == "linux-gnu"* || "$OSTYPE" == "linux-musl"* ]]; then
```

Comment:
Thank you for improving our OS detection in the CircleCI Orb.

While this is a great improvement, I'd take this a step further and adjust it so that we accept all Linux variants:
```suggestion
if [[ "$OSTYPE" == "linux"* ]]; then
```

Would this work for your use case as well? 🤔